internal/domain/filter.go

```diff
@@ -375,11 +375,11 @@ func (f Filter) CheckFilter(r *Release) ([]string, bool) {
 		r.addRejectionF("tags unwanted. got: %v want: %v", r.Tags, f.ExceptTags)
 	}
 
-	if len(f.Artists) > 0 && !containsFuzzy(r.TorrentName, f.Artists) {
+	if len(f.Artists) > 0 && !contains(r.Artists, f.Artists) {
 		r.addRejectionF("artists not matching. got: %v want: %v", r.TorrentName, f.Artists)
 	}
 
-	if len(f.Albums) > 0 && !containsFuzzy(r.TorrentName, f.Albums) {
+	if len(f.Albums) > 0 && !contains(r.Title, f.Albums) {
```

Comment:
Does Subtitle ever contain anything for tunes?

Reply:
Not that I know!